fastlane/lib/fastlane/lane_manager.rb

```diff
@@ -124,6 +138,9 @@ def self.choose_lane(ff, platform)
           end
 
           return platform, lane_name # yeah
+        else
+          UI.message "Run `fastlane` the next time you need to build, test or release your app :)"
```

Comment:
Yes to the emojis! And it only shows up when the table is printed as an opt out.